suite/src/handlers/test-api.ts

```diff
@@ -0,0 +1,47 @@
+import jest from 'jest'
+import * as url from 'url'
+import { configDotenv } from 'dotenv'
+
+import { utilities } from '../utils/common.js'
+import { createAnchorTable } from '../utils/dynamoDbHelpers.js'
+
+const failure = utilities.failure
+const success = utilities.success
+const JestArgs = process.env.JEST_ARGS ? process.env.JEST_ARGS.split(' ') : ['']
+
+// Load the .env file if it was specified
+if (process.env.ENV_PATH) {
+    configDotenv({path: process.env.ENV_PATH})
+}
+
+// Handles API requests for invoking (individual) tests directly via API, mainly for consumption in CI/CD pipelines but
+// can also be used for adhoc testing.
+export const handler: any = async (event: any) => {
+    console.log(JSON.stringify(event))
+    try {
+        // Make sure the test table exists before running tests
+        await createAnchorTable()
+        // Setup and run tests
+        const body = JSON.parse(event?.body ?? '{}')
```

Comment:
We can get rid of `event` and its parsing. This is a holdover from the AWS Lambda days when an event was being passed in to this handler.

Can also get rid of the `SUITE` envar since it doesn't seem to be very useful anymore, though it is a way to run specific test suites.